bundles/org.openhab.binding.speedtest/README.md

```diff
@@ -0,0 +1,92 @@
+# Speedtest Binding
+
+The Speedtest Binding can be used to perform a network speed test for your openHAB instance.
+It is based on the command line interface (CLI) version of Ookla's Speedtest (https://www.speedtest.net/apps/cli).
+
+The Ookla CLI Speedtest application MUST be installed on your openHAB instance when using the Speedtest Binding.
+
+
+## Why Ookla's Speedtest?
+
+Fully supported and maintained by Ookla with the latest Speedtest technology for best results:
+
+* Consistent measurement, even on high bandwidth
+* Consistent measurement, very much independent from the performance of the host system
+
+
+## What functionality does Ookla's Speedtest offer?
```

Comment:
Why list both what Ookla's speedtest can do and wat the binding can do? This might be confusing. From my point of view, it would be enough to just list the things the binding supports. Maybe with something like: 'While the Ookla's speedtest offers many more functions, the binding only support the following measures:'

Reply:
Please have a short look on my recent updates. I guess it makes the intention moe clear.
Do you still find it confusing?

Reply:
Sorry, but i'm not seeing it, can you elaborate?

Reply:
The binding is just a wrapper. The main functionality is provided by the 3rd party Oakla tool. This is also the biggest benefit of this binding compared to all the other solutions that are around.
The first paragraph lists the features from the Oakla tool, that are used by the binding.
The second paragraph lists the interfaces that the binding uses to provide this functionality to openHAB.
I hoped this get's clearer, using the updated description from https://github.com/openhab/openhab-addons/pull/9913/commits/6ee0a007e28f3a8325addcccd2eb0963769069d5

Reply:
Thank you, the doc improvement helped.